README.md

```diff
@@ -7,6 +7,8 @@ Notifications frontend for Red Hat Insights
 
 ## Build app
 
+0. If needed run `yarn install` to install dependency packages
+
 1. ```yarn```
```

Comment:
```suggestion
1. If needed run `yarn` or `yarn install` to install dependency packages
```

Reply:
I mention it like this, because I did a yarn and then yarn start as described. Yarn start failed. Then only as I gave yarn install it installed the dependencies and also env-cmd, which was blocking the yarn start startup.

Reply:
yarn should do an install

```bash
$ yarn
yarn install v1.22.17
[1/4] Resolving packages...
success Already up-to-date.
```

Reply:
Anyway is not a big deal to have that step in there